RadixWallet/Clients/FactorSourcesClient/FactorSourcesClient+Live.swift

```diff
@@ -254,8 +254,13 @@ extension FactorSourcesClient: DependencyKey {
 					)
 				).indices
 
-				guard let max = indices.max() else { return 0 }
-				let nextIndex = max + 1
+				guard let max = indices.max() else {
```

Comment:
so when implementing MFA, we would extend `nextEntityIndexForFactorSource` to also indicate whether the next index should be on the securified/unsecurified key space, correct?

Reply:
We will be using `FactorInstancesProvider` - https://github.com/radixdlt/sargon-mfa/pull/18/ either we will be deriving for Securified half or Unsecurified, correct.